merkato/merkato.py

```diff
@@ -2,24 +2,26 @@
 import json
 
 from merkato.exchanges.tux_exchange.exchange import TuxExchange
-from merkato.utils import create_price_data
-from merkato.constants import BUY, SELL, ID, PRICE, LAST_ORDER, ASK_RESERVE, BID_RESERVE, ASK_BUDGET, BID_BUDGET
-from merkato.utils.database_utils import update_merkato, get_all_merkatos_from_exchange
+from merkato.constants import BUY, SELL, ID, PRICE, LAST_ORDER, ASK_RESERVE, BID_RESERVE, EXCHANGE
+from merkato.utils.database_utils import update_merkato, insert_merkato
+from merkato.exchanges.tux_exchange.utils import translate_ticker
+from merkato.utils import create_price_data, validate_merkato_initialization, get_relevant_exchange
 from math import floor
 
 DEBUG = True
 
 
 class Merkato(object):
-    def __init__(self, configuration, exchange, coin, base, spread, ask_budget, bid_budget):
-        self.exchange = TuxExchange(configuration, coin=coin, base=base)
-        self.mutex_UUID = configuration['exchange'] + "coin={}_base={}".format(coin,base)
+    def __init__(self, configuration, coin, base, spread, UUID):
```

Comment:
I dont think the user(GUI) will know the UUID format, I think it should still be calculated in init

Reply:
it's derived information

Reply:
sure